src/js/flows/submitSearch.js

```diff
@@ -26,9 +26,16 @@ export default function submitSearch(
 
     const state = getState()
 
+    let view = document.getElementsByClassName("view")[0]
```

Comment:
When using react, it's considered good practice to avoid grabbing an element like this. Instead, they would say set some state (like `Viewer.setScroll`), then have the component containing that element "react" to the state change.

And to get the x, y values. Set the state in the onScrollEnd callback in the Viewer.

This declarative style of building UI makes reasoning about things easier. However there are situations when it makes sense to break out of the react paradigms and use DOM methods like this.